segmentstore/server/src/main/java/io/pravega/segmentstore/server/logs/operations/StreamSegmentMapOperation.java

```diff
@@ -115,19 +133,20 @@ public boolean isSealed() {
     @Override
     public String toString() {
         return String.format(
-                "%s, Id = %s, Name = %s, Start = %d, Length = %d, Sealed = %s",
+                "%s, Id = %s, Name = %s, Start = %d, Length = %d, Sealed = %s, Pinned = %s",
                 super.toString(),
                 toString(getStreamSegmentId(), ContainerMetadata.NO_STREAM_SEGMENT_ID),
                 getStreamSegmentName(),
                 getStartOffset(),
                 getLength(),
-                isSealed());
+                isSealed(),
+                isPinned());
     }
 
     //endregion
 
     static class Serializer extends OperationSerializer<StreamSegmentMapOperation> {
-        private static final int STATIC_LENGTH = 4 * Long.BYTES + Byte.BYTES;
+        private static final int STATIC_LENGTH = 4 * Long.BYTES + 2 * Byte.BYTES;
```

Comment:
Should we not make this compatible with `0.4`?

Reply:
We can, but it wouldn't provide any benefit. That would make the serialization backwards compatible at the expense of more bytes written with every serialization and more CPU used for every (de)serialization. However this is part of a larger change which is not backwards compatible due to the Segment State changing storage media from Tier 2 to a Table Segment. 

Both the previous PR for this change (#3206 - merged ) and the following one (#3150 not yet submitted) introduced or will introduce such changes.